### DISCUSSION TITLE
what resources to go through for contribution

### DISCUSSION THREAD

**Discussion by six-eyes-dev** [cncf/glossary#1711] [Q&A]
Would anyone please suggest me some resources I should go through for start contributing.

**Comment by SaxenaAnushka102** [comment]
Hi @himanshusingh0905! You can start with the [Contributing guide](https://github.com/cncf/glossary/blob/main/CONTRIBUTING.md) & [Localization guide](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md).
Also, make sure to follow the [Code of Conduct](https://github.com/cncf/glossary/blob/main/code-of-conduct.md) while contributing to the community. Let's collaborate & learn!!
